src/main/java/uk/gov/pay/adminusers/resources/service/ServiceRequestValidatorV2.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.pay.adminusers.resources.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
As discussed sounds like we're agreed it might be better to avoid temp new versions to handle JsonArrays and you think we can wrap a JsonNode into a JsonArray to maintain backwards compatibility and then have a common process from there on in. I'll stop the review there to avoid commenting on things that may disappear.